pmsoService/product_flow/features/import_request/api/views/list_create_views.py

```
@@ -0,0 +1,115 @@
+        if page is not None:
+            prefetch_transactions_for_imports(page)
+            serializer = self.get_serializer(page, many=True)
+            return self.get_paginated_response(serializer.data)
+        
+        # If no pagination, prefetch for all results
+        import_requests = list(queryset)
+        prefetch_transactions_for_imports(import_requests)
+        serializer = self.get_serializer(import_requests, many=True)
+        return Response(serializer.data)
+
+    def create(self, request, *args, **kwargs):
```

Ruff (0.13.3)

Unused method argument: `args`

(ARG002)